Cargo.toml

```diff
@@ -29,16 +29,14 @@ cranelift = [ "cosmwasm-vm/default-cranelift"]
 singlepass = [ "cosmwasm-vm/default-singlepass"]
 
 [dependencies]
-cosmwasm = { version = "~0.6.0" }
-cw-storage = "0.1.0"
-named_type = "=0.2.1"
-named_type_derive = "=0.2.1"
+cosmwasm = { version = "~0.7.0" }
```

Comment:
Independent of this PR, but when you are working on this now:

Can you change this to `version = "0.7.0"`, which in Rust implies `"^0.7.0"`. This includes all compatible patch releases.

Reply:
should i leave off the last zero or is it wanted to specifically get .7.0?

Reply:
`version = "0.7.0"` and `version = "0.7"` should be the same when it comes to matching versions. I'd keep the last zero, but don't have a strong opinion here.

Reply:
if its 0.7 does it not pull the most recent version, but if you specify 0.7.0 it will always fetch that version

Reply:
The version is fixed in `Cargo.lock`. If you specify `0.7.0` in `Cargo.toml`, it means when updating the lockfile pull semver range `^0.7.0`, which is `>= 0.7.0` and `< 0.8.0`.

So `0.7.0` in `Cargo.toml` can be `0.7.5` in `Cargo.lock`.

If you prefix a `=` instead of the implicit `^`, you really fix the exact version.

Reply:
ah okay thank you for the explanation